site/en/docs/extensions/reference/commands/index.md

```diff
@@ -134,10 +134,8 @@ chrome.commands.onCommand.addListener((command) => {
 
 ### Action commands
 
-The `_execute_action` (Manifest V3), `_execute_browser_action` (Manifest V2), and
-`_execute_page_action` (Manifest V2) commands are reserved for the action of trigger your action,
-browser action, or page action respectively. These commands do not dispatch
-[command.onCommand][event-oncommand] events like standard commands.
+The `_execute_action` command is reserved for triggering your action,
+it does not dispatch [`command.onCommand`][event-oncommand] events like standard commands.
```

Comment:
```suggestion
it does not dispatch [`command.onCommand`][event-oncommand] events as do standard commands.
```

Reply:
The distinction between 'like' and 'as' is one that rarely causes comprehension difficulties. In this particular case, I think it might for non-native readers.